01week/rockPaperScissors.js

```diff
@@ -8,10 +8,28 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+function rockPaperScissors(arg1, arg2) {
+  
+  const hand1 = arg1.trim().toLowerCase();
+  const hand2 = arg2.trim().toLowerCase();
+  //console.log(hand1,hand2)
 
-  // Write code here
+  if (hand1===hand2){
+    return "It's a tie";
+  }else if
+  (hand1==='rock' && hand2==='scissors'||
+  hand1==='scissors' && hand2==='paper'||
+hand1==='paper' && hand2==='rock'){
+    return "Hand one wins!";
+
+  }else if(hand2==='paper' && hand1==='rock'||
```

Comment:
Great work on your logic.  However, consider refactoring this code so that you are not repeating yourself.  Your order of conditions are great, but rethink another way you could accomplish returning a win for hand 2

Reply:
I'm still brainstorming on this issue. Thanks for your help.